tests/integration/month.rs

```diff
@@ -47,3 +49,37 @@ fn display() {
     assert_eq!(November.to_string(), "November");
     assert_eq!(December.to_string(), "December");
 }
+
+#[test]
+fn to_u8() {
+    assert_eq!(u8::from(January), 1u8);
+    assert_eq!(u8::from(February), 2u8);
+    assert_eq!(u8::from(March), 3u8);
+    assert_eq!(u8::from(April), 4u8);
+    assert_eq!(u8::from(May), 5u8);
+    assert_eq!(u8::from(June), 6u8);
+    assert_eq!(u8::from(July), 7u8);
+    assert_eq!(u8::from(August), 8u8);
+    assert_eq!(u8::from(September), 9u8);
+    assert_eq!(u8::from(October), 10u8);
+    assert_eq!(u8::from(November), 11u8);
+    assert_eq!(u8::from(December), 12u8);
+}
+
+#[test]
+fn try_from_u8() {
+    assert!(Month::try_from(0u8).is_err());
```

Comment:
It would be good to assert the component name here.

```suggestion
    assert!(matches!(Month::try_from(0u8), Err(err) if err.name() == "month"));
```